WcaOnRails/spec/controllers/registrations_controller_spec.rb

```diff
@@ -161,6 +161,15 @@
       patch :update, id: registration.id, registration: { status: :accepted }
       expect(registration.reload.accepted?).to eq true
     end
+
+    it "cannot accept own registration for a competition that isn't visible" do
+      competition.showAtAll = 0
+      competition.save!
```

Comment:
Do we need to run the validations? I think we can just update the table directly. We have something similar [here](https://github.com/cubing/worldcubeassociation.org/blob/master/WcaOnRails/spec/controllers/competitions_controller_spec.rb#L90-L96).